helix-core/src/test/java/org/apache/helix/manager/zk/TestZkBaseDataAccessor.java

```diff
@@ -433,6 +433,36 @@ public void testSyncRemove() {
     System.out.println("END " + testName + " at " + new Date(System.currentTimeMillis()));
   }
 
+  @Test
+  public void testDeleteLogging() {
+    String root = _rootPath;
+
+    ZkBaseDataAccessor<ZNRecord> accessor = new ZkBaseDataAccessor<>(_gZkClient);
```

Comment:
Thanks for writing up the test. 
Logging test can be tricky, it depends on the configuration of log level in test environment, and there isn't a good way to verify the log message. 
If I understand correctly, this test is only to test logging, but not to test the delete functionality?

Reply:
Correct, this is just to check that the specific logging message is displayed. I believe delete functionality is tested in `TestZkBaseDataAccessor.testAsyncZkBaseDataAccessor()`

Reply:
Thanks. In this case, I don't think we need this test at all.

Reply:
I might have phrased my answer wrong. The test does check whether zk catches the error of removing a parent node with children, which I believe is not tested elsewhere, though not sure if that is considered as delete functionality. The primary motive for the test is to catch this error and make sure the right logging is made. Perhaps that is not necessary given the variable nature of log configs - let me know, thanks!

Reply:
Ok, I see your point. My concern is that, `"make sure the right logging is made"` is checked manually. It doesn't add much value to the unit test in regard to logging. Functionally, it does have value to verify no exception is thrown. So we may keep the test, and rename it to e.g. `testDeleteNodeWithChildren`, just my two cents.

Reply:
I agree with renaming the tests for better clarity - thanks for the feedback!